sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -539,15 +539,15 @@ case class DescribeTableCommand(
         throw new AnalysisException(
           s"DESC PARTITION is not allowed on a temporary view: ${table.identifier}")
       }
-      describeSchema(catalog.lookupRelation(table).schema, result, header = false)
+      describeSchema(catalog.lookupRelation(table).schema, result, header = true)
```

Comment:
May be should add a configure like `hive.cli.print.header`.

Reply:
It's not worth yet another flag, and this is a behavior change in the output. There's no motivation for this change either.

Reply:
We did it on purpose. For details, see https://github.com/apache/spark/pull/18203

Reply:
# Partition Information
# col_name      data_type       comment

Partition information also takes up two rows.
I try to keep the head of the case, let rows number is displayed correctly.

Reply:
The snapshot is correct fix code effect, the statistics rows does not contain the head
![2](https://user-images.githubusercontent.com/26266482/36069344-ba833c56-0f22-11e8-9ab6-26f0ae6285b7.png)

Reply:
@gatorsmile I wasn't clear -- you're saying this change is _wrong_ because it re-adds a header that was removed in the other PR?

Reply:
Yeah. Hive beeline already hardcoded the header. See the https://issues.apache.org/jira/browse/SPARK-20954